src/lib/three.js

```diff
@@ -1,11 +1,27 @@
-var THREE = require('./three.mjs').default;
```

Comment:
Since we're now using modules, there is no need for this file to act as an intermediate for `three.mjs`.